war/src/main/js/plugin-manager-ui.js

```diff
@@ -4,8 +4,6 @@ import requestAnimationFrame from 'raf';
 import pluginManagerAvailable from './templates/plugin-manager/available.hbs'
 import pluginManager from './api/pluginManager';
 
-var filterInput = document.getElementById('filter-box');
```

Comment:
this relied on the javascript being loaded in the page after the input was defined, moved it to only try access it when it exists so moving the input around doesn't break the JS

Reply:
Okay